src/lib.rs

```diff
@@ -194,6 +194,16 @@ impl<T: Any> Downcast for T {
     fn as_any_mut(&mut self) -> &mut dyn Any { self }
 }
 
+pub trait DowncastSend: Downcast + Send {
+    fn into_any_send(self: Box<Self>) -> Box<dyn Any + Send>;
```

Comment:
It feels a little less elegant that
1. This method is not supported for `DowncastSync` given that it also includes `Downcast + Send` in its constraints.
2. We're not supporting `into_any_sync`. Do you anticipate no need for this?

I suppose we could publish the current change that you've written and then conceivably do both of the above changes and then publish a 2.0 version. I'm not sure how much that's worth it. Thoughts?

Reply:
Good catch! In my mind if you wanted a `Box<dyn Any + Send>` you'd just use `DowncastSend` and not expect `DowncastSync` to provide it for you, but I can get why this might be wanted.

I'll investigate if adding the supertrait is a breaking change at all